@dpaz Build fails:
```
# github.com/bblfsh/tools/cmd/bblfsh-tools

cmd/bblfsh-tools/cyclomatic.go:10: undefined: tools.CyclomaticComp

make: *** [github.com/bblfsh/tools/cmd/bblfsh-tools] Error 2
```

--- a/cmd/bblfsh-tools/cyclomatic.go
+++ b/cmd/bblfsh-tools/cyclomatic.go
@@ -7,5 +7,5 @@ type CyclomaticComp struct {
 }
 
 func (c *CyclomaticComp) Execute(args []string) error {
-	return c.execute(args, tools.CyclomaticComp{})
+	return c.execute(args, tools.CyclomaticComplexity{})
 }

--- a/cyclomatic.go
+++ b/cyclomatic.go
@@ -46,13 +46,13 @@ import (
 type CyclomaticComplexity struct{}
 
 func (cc CyclomaticComplexity) Exec(n *uast.Node) error {
-	result := CyclomaticComplex(n)
-	fmt.Println("Cyclomatic Complexity= ", result)
+	result := cyclomaticComplexity(n)
+	fmt.Println("Cyclomatic Complexity = ", result)
 	return nil
 }
 
-func CyclomaticComplex(n *uast.Node) int {
-	complx := 1
+func cyclomaticComplexity(n *uast.Node) int {
+	complexity := 1
 
 	iter := uast.NewOrderPathIter(uast.NewPath(n))
 
@@ -64,11 +64,11 @@ func CyclomaticComplex(n *uast.Node) int {
 		n := p.Node()
 		for _, r := range n.Roles {
 			if addsComplexity(r) {
-				complx++
+				complexity++
 			}
 		}
 	}
-	return complx
+	return complexity
 }
 
 func addsComplexity(r uast.Role) bool {

--- a/cyclomatic_test.go
+++ b/cyclomatic_test.go
@@ -28,5 +28,5 @@ func TestCyclomaticComplexity(t *testing.T) {
 				{InternalType: "break", Roles: []uast.Role{uast.Break}},
 			},
 			}}}
-	require.Equal(CyclomaticComplex(n), 6)
+	require.Equal(cyclomaticComplexity(n), 6)
 }

--- a/glide.lock
+++ b/glide.lock
@@ -1,5 +1,5 @@
-hash: d15f340e9e4663fd5c63232d33fbf9816fbe0a89ded1d61d99f86e48e9c8c417
-updated: 2017-06-01T18:27:35.508534629+02:00
+hash: a81eeb5ad7dde7d1e106bc20f11ce13ece6097d78dfd1cf502c4d65b57345ffa
+updated: 2017-06-06T11:54:07.833141099+02:00
 imports:
 - name: github.com/bblfsh/sdk
   version: 2ee6544d9c7dcf1bc4fac98e4e0f565ea3a92402
@@ -66,4 +66,17 @@ imports:
   - transport
 - name: srcd.works/go-errors.v0
   version: 6f07baca276330431b44ece8ee84ac98167bc4ea
-testImports: []
+testImports:
+- name: github.com/davecgh/go-spew
+  version: 6d212800a42e8ab5c146b8ace3490ee17e5225f9
+  subpackages:
+  - spew
+- name: github.com/pmezard/go-difflib
+  version: d8ed2627bdf02c080bf22230dbb337003b7aba2d
+  subpackages:
+  - difflib
+- name: github.com/stretchr/testify
+  version: 69483b4bd14f5845b5a1e55bca19e954e827f1d0
+  subpackages:
+  - assert
+  - require

--- a/glide.yaml
+++ b/glide.yaml
@@ -10,3 +10,9 @@ import:
   version: v1.2.0
 - package: google.golang.org/grpc
   version: v1.3.0
+- package: srcd.works/go-errors.v0
+testImport:
+- package: github.com/stretchr/testify
+  version: v1.1.4
+  subpackages:
+  - require

